mods/persistence/controllers/subsystems/persistence.dm

```diff
@@ -460,4 +394,16 @@
 
 /datum/controller/subsystem/persistence/show_info(mob/user)
 	to_chat(user, SPAN_INFO("Disabled with persistence modpack (how ironic)..."))
-	return
\ No newline at end of file
+	return
+
+/datum/controller/subsystem/persistence/proc/AddToLimbo(var/datum/thing, var/key, var/limbo_type, var/metadata, var/modify = TRUE)
+	return one_off.AddToLimbo(thing, key, limbo_type, metadata, modify)
+
+/datum/controller/subsystem/persistence/proc/RemoveFromLimbo(var/limbo_key, var/limbo_type)
+	return one_off.RemoveFromLimbo(limbo_key, limbo_type)
+
+/datum/controller/subsystem/persistence/proc/DeserializeOneOff(var/limbo_key, var/limbo_type, var/remove_after = TRUE)
+	return one_off.RemoveFromLimbo(limbo_key, limbo_type, remove_after)
```

Comment:
DeserializeOneOff here, I'm guessing, not RemoveFromLimbo?

Reply:
That's a good catch!